includes/class-indieauth-authenticate.php

```diff
@@ -92,29 +96,33 @@ public static function authorization_redirect( $me, $redirect_uri ) {
 		$query = build_query(
 			array(
 				'me'           => rawurlencode( $me ),
-				'redirect_uri' => wp_login_url( $redirect_uri ),
-				'client_id'    => home_url(),
+				'redirect_uri' => rawurlencode( $redirect_uri ),
+				'client_id'    => rawurlencode( home_url() ),
 				'state'        => wp_create_nonce( 'indieauth-' . home_url() ),
+				'response_type' => 'id', 
+				'scope'        => '*',  // We will consider this a global scope
```

Comment:
For `response_type=id`, scope is not part of the request. Scope is only used for authorization (`response_type=code`).